addons/account/models/account_account.py

```diff
@@ -198,8 +198,10 @@ def _check_used_as_journal_default_debit_credit_account(self):
         self._cr.execute('''
             SELECT journal.id
             FROM account_journal journal
+            LEFT JOIN account_payment_method_line apml ON journal.id = apml.journal_id
```

Comment:
I'm pretty sure account payment methods arrived in 14.4? so it shouldn't be there before 15.0 :)

Also, the task linked to the commit message seems to be wrong

Reply:
Indeed, I was testing on `15.0` and did not check when I copy/pasted the code in `14.0`, lesson learned :sweat_smile: 
And I also corrected the link :wink: 
Thanks for the feedback :slightly_smiling_face: